tests/schedulers/airflow/operators/test_hwm_sensor.py

```diff
@@ -13,10 +15,17 @@
 )
 from sqlmesh.utils.date import to_datetime
 
+pytest_plugins = ["tests.schedulers.airflow.operators.fixtures"]
 pytestmark = pytest.mark.airflow
 
 
-def test_no_current_hwm(mocker: MockerFixture, make_snapshot, random_name):
+@pytest.fixture
+def set_airflow_as_library():
```

Comment:
Can we somehow have a single copy of this fixture? Maybe put it into `.../operators/conftest.py`?

Reply:
This is a typo the single copy is in `tests/schedulers/airflow/operators/fixtures.py` since it is just used in here